src/pages/Game.js

```diff
@@ -29,12 +35,16 @@ export const Game = () => {
     <div className='App'>
       <header>
         <h1>Tic-Tac-Toe</h1>
-        <div className='user'>
-          <Button variant='contained' size='small'>
-            Change
-          </Button>
-        </div>
       </header>
+      <br />
+      <div className='finishedGame'>
+        {isFinished && winner != null ? (
```

Comment:
We don't have to use the ternary operator here. We can also do:
`{isFinished && winner !== null && <Alert... >}`